main_test.go

```diff
@@ -51,17 +50,6 @@ func newTestAutographer(t *testing.T) (*autographer, configuration) {
 	}
 
 	t.Cleanup(func() {
-		host := database.GetTestDBHost()
```

Comment:
This is unrelated change and I worry about having to put this right back after we add more tests.